catalog/MDCCatalog/MDCMenuViewController.swift

```diff
@@ -23,13 +23,29 @@ import MaterialComponents.MaterialLibraryInfo
 
 class MDCMenuViewController: UITableViewController {
 
-  let tableData =
-    [(title: "Settings", icon: MDCIcons.imageFor_ic_settings()?.withRenderingMode(.alwaysTemplate)),
-     (title: "Themes", icon: MDCIcons.imageFor_ic_color_lens()?.withRenderingMode(.alwaysTemplate)),
-     (title:  "v\(MDCLibraryInfo.versionString)",
-      icon: MDCIcons.imageFor_ic_help_outline()?.withRenderingMode(.alwaysTemplate))]
+  private struct MDCMenuItem {
+    let title: String!
+    let icon: UIImage?
+    let accessibilityLabel: String?
+    let accessibilityHint: String?
+    init(_ title: String, _ icon: UIImage?, _ accessibilityLabel: String?,
+         _ accessibilityHint: String?) {
+      self.title = title
+      self.icon = icon
+      self.accessibilityLabel = accessibilityLabel
+      self.accessibilityHint = accessibilityHint
+    }
+  }
+
+  private let tableData =
+    [MDCMenuItem("Settings", MDCIcons.imageFor_ic_settings()?.withRenderingMode(.alwaysTemplate),
+                 nil, "Opens debugging menu."),
+     MDCMenuItem("Themes", MDCIcons.imageFor_ic_color_lens()?.withRenderingMode(.alwaysTemplate),
+                  nil, "Opens color theme chooser."),
+     MDCMenuItem("v\(MDCLibraryInfo.versionString)",
+      MDCIcons.imageFor_ic_help_outline()?.withRenderingMode(.alwaysTemplate),
+      "Version \(MDCLibraryInfo.versionString)", "Closes this menu.")]
```

Comment:
Can we make this optional based on whether VO is on?
This is a non-blocking question, but does impact the design of our 'showroom' app.

Reply:
I'm not sure I understand the suggestion, I believe this change is only for `accessibilityLabel` and `accessibilityHint`. Maybe I accidentally changed something else? Could you please be a little more specific?

Reply:
Discussed Offline.